<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transformer</class>
 <widget class="QWidget" name="Transformer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video transformer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="4">
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string>Progress</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>298</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="addPushButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="delPushButton">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="startPushButton">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Video codec:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Audio codec:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="vcodecComboBox">
         <item>
          <property name="text">
           <string>mpeg4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>xvid</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>divx</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="widthSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>2000</number>
         </property>
         <property name="value">
          <number>480</number>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QComboBox" name="acodecComboBox">
         <item>
          <property name="text">
           <string>mp3lame</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ac3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>faac</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Height:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Quality:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QSpinBox" name="qualitySpinBox">
         <property name="toolTip">
          <string>1 - 31. 1 means best.</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>31</number>
         </property>
         <property name="value">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QSpinBox" name="heightSpinBox">
         <property name="toolTip">
          <string>-3 means auto.</string>
         </property>
         <property name="minimum">
          <number>-3</number>
         </property>
         <property name="maximum">
          <number>1500</number>
         </property>
         <property name="value">
          <number>-3</number>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Audio rate:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QComboBox" name="srateComboBox">
         <item>
          <property name="text">
           <string>22050</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>44100</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="2" colspan="4">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
